README.md

```diff
@@ -218,6 +226,124 @@ func userNotificationCenter(
+2. **URL schemes** - In the URL Schemes box, specify the prefix you use for your URLs.
+3. **Role** - Since your app will be editing the role select the role as editor
+
+In order to edit the Info.plist directly, just fill in your app specific details and paste this in your plist - 
+
+```xml
+<key>CFBundleURLTypes</key>
+<array>
+	<dict>
+		<key>CFBundleTypeRole</key>
+		<string>Editor</string>
+		<key>CFBundleURLName</key>
+		<string>{your_unique_identifier}</string>
+		<key>CFBundleURLSchemes</key>
+		<array>
+			<string>{your_URL_scheme}</string>
+		</array>
+	</dict>
+</array>
+```
+
+
+### Step 2: Whitelist supported URL schemes
+
+Since iOS 9 Apple has mandated that the URL schemes that you app can open need to also be listed in the Info.plist. This is in addition to Step 1 above. Even if your app isn't opening any other apps, you still need to list your app's URL scheme in order for deep linking to work. 
+
+This needs to be done in the Info.plist directly - 
+
+```xml
+<key>LSApplicationQueriesSchemes</key>
+<array>
+	<string>{your custom URL scheme}</string>
+</array>
+```
+
+### Step 3: Implement handling deep links in your app
+
+Steps 1 & 2 set your app up for receiving deep links but now is when you need to figure out how to handle them within your app. 
+
+If you are using UIKit, you need to implement [`application:openURL:options:`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623112-application) in your application's app delegate.
+
+Example: 
+
+```swift
+func application(
+    _ app: UIApplication,
+    open url: URL,
+    options: [UIApplication.OpenURLOptionsKey : Any] = [:]
+) -> Bool {
+    guard let components = NSURLComponents(url: url, resolvingAgainstBaseURL: true),
+          let host = components.host
+    else {
+       print("Invalid deep linking URL")
+       return false
+    }
+                   
+    print("components: \(components.debugDescription)")
+   
+    // Create the deep link
+    guard let deeplink = DeepLinking(rawValue: host) else {
+        print("Deeplink not found: \(host)")
+        return false
+    }
+    
+    // implement your custom logic here
+    handle(deeplink, with: url.description)
+    
+    return true
+}
+```
+
+If you are using SwiftUI, then you can implement [`onOpenURL(perform:)`](https://developer.apple.com/documentation/swiftui/view/onopenurl(perform:)) as a view modifier in the view you intent to handle deep links. This may or may not be the root of your scene 
+
+Example: 
+
+```swift
+@main
+struct MyApplication: App {
+  var body: some Scene {
+    WindowGroup {
+      ContentView()
+        .onOpenURL { url in
+          // handle the URL that must be opened
+        }
+    }
+  }
+}
+``` 
+
+Once the above steps are complete, you can now try to send push notifications from the Klaviyo Push editor (refer example below) on the web and make sure that the URL shows up in the handler you implemented in Step 3.
```

Comment:
Another rewording suggestion:
`Once the above steps are complete, you can try to send push notifications from the Klaviyo Push editor within the Klaviyo website. Here you can build and send a push notification through Klaviyo to make sure that the URL shows up in the handler you implemented in Step 3 once it has been extracted from the received push notification payload.`

Reply:
Done. Except I didn't add the last sentence - "once it has been extracted from the received push notification payload" since we now handle it on the user's behalf.
https://klaviyo.slack.com/archives/C042EUAGAMC/p1677776693908989